xmake/modules/package/tools/cmake.lua

```diff
@@ -1214,8 +1220,9 @@ end
 -- install package
 function install(package, configs, opt)
     opt = opt or {}
-    local cmake_generator = _get_cmake_generator(package, opt)
+    _get_configs_for_generator(package, configs, opt)
```

Comment:
we should not call `_get_configs_for_generator `, 

please detect and set ninja generator in _get_cmake_generator, and remove

https://github.com/xmake-io/xmake/blob/2a2ee54fe65cd4035bc25b01391e63170c952c38/xmake/modules/package/tools/cmake.lua#L760-L766